spec-main/fixtures/crash-cases/webview-contents-error-on-creation/index.js

```diff
@@ -0,0 +1,47 @@
+// Modules to control application life and create native browser window
+const { app, BrowserWindow } = require('electron');
+const path = require('path');
+
+app.on('web-contents-created', (wc) => {
+  throw new Error();
+});
+
+function createWindow () {
+  // Create the browser window.
+  const mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      preload: path.join(__dirname, 'preload.js')
+    }
+  });
+
+  // and load the index.html of the app.
+  mainWindow.loadFile('index.html');
+
+  // Open the DevTools.
+  // mainWindow.webContents.openDevTools()
+}
+
+// This method will be called when Electron has finished
+// initialization and is ready to create browser windows.
+// Some APIs can only be used after this event occurs.
+app.whenReady().then(() => {
+  createWindow();
+
+  app.on('activate', function () {
+    // On macOS it's common to re-create a window in the app when the
+    // dock icon is clicked and there are no other windows open.
+    if (BrowserWindow.getAllWindows().length === 0) createWindow();
+  });
+});
+
+// Quit when all windows are closed, except on macOS. There, it's common
+// for applications and their menu bar to stay active until the user quits
+// explicitly with Cmd + Q.
+app.on('window-all-closed', function () {
+  if (process.platform !== 'darwin') app.quit();
+});
+
+// In this file you can include the rest of your app's specific main process
+// code. You can also put them in separate files and require them here.
```

Comment:
Let's simplify this to the bare minimum to make it obvious what's being tested. Also, let's exit the process with a 0 exit code when we're successful.

```suggestion
const { app, BrowserWindow } = require('electron');

app.on('web-contents-created', (wc) => {
  throw new Error();
});

app.whenReady().then(async () => {
  const mainWindow = new BrowserWindow({ show: false });
  await mainWindow.loadURL('about:blank');
  process.exit(0);
})
```

Reply:
Oof, yes, thank you for catching that 🙇‍♀️